proto/canal/EntryProtocol.pb.go

```diff
@@ -84,6 +84,8 @@ const (
 	EventType_XAROLLBACK EventType = 14
 	//* MASTER HEARTBEAT *
 	EventType_MHEARTBEAT EventType = 15
+	//* TiCDC CUSTOMIZED *
+	EventType_WATERMARK EventType = 16
```

Comment:
What if `canal` add another filed = 16 in the future?
Besides is there any side effect to add one more field, such as will it break the usage of existing consumer.

Reply:
Can you give me any hint about how to handle the situation mentioned above ?

Maybe we can define `WATERMARK` as a constant, without altering the protobuf related files?

For the existing consumers, it will not receive any message containing this event type, because we controlled by a parameter `waterMark` default to false. 

For any user that wants to use this event type, we should provide concrete documentation and consumer example, which will be done in the near future.